### PR TITLE
update license

### PR DESCRIPTION
**Pull request by bdangit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,11 @@ Then to run tests:
...
 ## Author and License
-Simple Finance <ops@simple.com>
+Simple Finance <ops@simple.com> - Original
 
 Apache License, Version 2.0
+
+## Maintainers
+Ben Dang <me@bdang.it>
+
+E Camden Fisher
```
Comment (fishnix) [inline README.md]: <fish@fishnix.net>

**Comment by fishnix** [comment]
This looks good to me.  Are we going to keep SimpleFinance as the copyright holder?  I'm not exactly sure how that works now that it's been adopted. :+1:

**Comment by bdangit** [comment]
since the code is fully transferred, we are probably within legal rights to modify the copyrights and licensing.  Apache 2.0 to MIT?

**Comment by fishnix** [comment]
@bdangit IANAL but it has a fairly open license AND you have been transferred ownership, so probably :question:   Maybe @jakedavis can come in here and give his :+1:

**Comment by miah** [comment]
+1

**Comment by jakedavis** [comment]
Yeah I mean ... I am also not a lawyer but like, sounds fine.

**Comment by bdangit** [comment]
Thanks @jakedavis / @miah .  I'll take your comments as an OK to make this code even more free.

Changed license to MIT.  Can I have your thumbs up @fishnix?

**Comment by fishnix** [comment]
looks good :+1:

**Pull request merged**